collect_app/src/main/res/values/strings.xml

```diff
@@ -507,4 +507,7 @@
     <string name="not_supported_offline_layer_format">Selected offline layer file uses the PBF format which is not supported!</string>
     <string name="sort_by">Sort by</string>
     <string name="server">Server</string>
+    <string name="form_delete_message">Deleting selected forms</string>
+    <string name="deleting_form_dialog_first">Deleting form:\u0020</string>
```

Comment:
The dialog message should be a single string and should take in the current and total number of forms as parameters. This is important because not all languages are left-to-right and translators must have the option of repositioning the two form counts relative to each other.